### ISSUE TITLE
AttributeError: 'unicode' object has no attribute 'items' when calling console.incidents.all()

### ISSUE THREAD

**Issue by jmvazquez** [thinkst/canarytools-python#7]
Using example in the API docs, I ran into an error when calling console.incidents.all()

On MacOS 10.15.2, Python version 2.7.16

Code I ran:

> console = canarytools.Console(api_key='excised api key', domain='excised domain')
> 
> console.incidents.all()

```Traceback (most recent call last):
  File "canary.py", line 4, in <module>
    for incident in console.incidents.all():
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 29, in all
    return self.console.get('incidents/all', params, self.parse)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/console.py", line 158, in get
    return self.handle_response(resp.json(), parser)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/console.py", line 219, in handle_response
    return parser(response)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 184, in parse
    self.console, incident))
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/base.py", line 10, in parse
    return cls(console, data)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 220, in __init__
    super(Incident, self).__init__(console, data)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/base.py", line 23, in __init__
    setattr(self, attribute, value)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 243, in __setattr__
    self.__setattr__(attribute, val)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 237, in __setattr__
    events.append(Event.parse(self.console, event))
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/base.py", line 10, in parse
    return cls(console, data)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/incidents.py", line 369, in __init__
    super(Event, self).__init__(console, data)
  File "/Users/jmvazque/Library/Python/2.7/lib/python/site-packages/canarytools/models/base.py", line 22, in __init__
    for attribute, value in sorted(data.items()):
AttributeError: 'unicode' object has no attribute 'items'```

**Comment by jmvazquez** [comment]
Same thing with Python 3, running in a clean sandbox VM:

Using this script:

`import canarytools
from configparser import ConfigParser

api_key,domain = getAPITokens()
console = canarytools.Console(api_key=api_key, domain=domain)
all_incidents = console.incidents.all()
`

`debian@debian:~/Desktop/canary$ python3 thinkst.py 
Traceback (most recent call last):
  File "thinkst.py", line 18, in <module>
    all_incidents = console.incidents.all()
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 29, in all
    return self.console.get('incidents/all', params, self.parse)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/console.py", line 158, in get
    return self.handle_response(resp.json(), parser)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/console.py", line 219, in handle_response
    return parser(response)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 184, in parse
    self.console, incident))
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/base.py", line 10, in parse
    return cls(console, data)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 220, in __init__
    super(Incident, self).__init__(console, data)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/base.py", line 23, in __init__
    setattr(self, attribute, value)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 243, in __setattr__
    self.__setattr__(attribute, val)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 237, in __setattr__
    events.append(Event.parse(self.console, event))
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/base.py", line 10, in parse
    return cls(console, data)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/incidents.py", line 369, in __init__
    super(Event, self).__init__(console, data)
  File "/home/debian/.local/lib/python3.7/site-packages/canarytools/models/base.py", line 22, in __init__
    for attribute, value in sorted(data.items()):
AttributeError: 'str' object has no attribute 'items'
`

**Comment by jayjb** [comment]
Hi @jmvazquez,

Thanks for reporting this. Ill take a look and get back to you.

**Comment by nickrohrbs** [comment]
This issue has been fixed in fb572f0
